doc/guides/writing-tests.md

```diff
@@ -193,16 +193,16 @@ const http = require('http');
 
 const server = http.createServer(common.mustCall((req, res) => {
   res.end();
-})).listen(0, function() {
+})).listen(0, common.mustCall(() => {
```

Comment:
BTW, there's no equivalent of this `common.mustCall` in the preceding code block. Should I add a corresponding `listening` counter and then increment it?

Reply:
I did that.